build-logic/jvm/src/main/kotlin/build-logic.test-junit5.gradle.kts

```diff
@@ -0,0 +1,22 @@
+import com.github.vlsi.gradle.dsl.configureEach
+
+plugins {
+    `java-library`
+    id("build-logic.test-base")
+}
+
+dependencies {
+    testImplementation("org.junit.jupiter:junit-jupiter-api")
+    testImplementation("org.junit.jupiter:junit-jupiter-params")
+}
+
+tasks.configureEach<Test> {
+    useJUnitPlatform()
```

Comment:
useJUnitPlatform resets all previous configuration for junit, check first if it was already applied and re-configure the existing TestFramework

Reply:
Looks like this nasty bug was fixed with gradle 8.x :tada: 
But we still don't need it as we already apply junitjacoco plugin which already takes care of this.

Reply:
I removed build-logic.test-* files